Formula/libsigrokdecode.rb

```diff
@@ -0,0 +1,54 @@
+class Libsigrokdecode < Formula
+  desc "Shared library providing protocol decoding functionality"
+  homepage "https://sigrok.org/wiki/Libsigrokdecode"
+  url "https://sigrok.org/download/source/libsigrokdecode/libsigrokdecode-0.5.3.tar.gz"
+  sha256 "c50814aa6743cd8c4e88c84a0cdd8889d883c3be122289be90c63d7d67883fc0"
+
+  # glib-2.0 >= 2.32.0
```

Comment:
Please remove the comment. If that's the current version in homebrew it won't ever be lower.